### ISSUE TITLE
Does latest magic sdk for react native support bare RN workflow?

### ISSUE THREAD

**Issue by ilyavf** [magiclabs/magic-js#369]
### ✅ Prerequisites

- [x] Did you perform a cursory search of open issues? Is this question already asked elsewhere?
- [x] Are you reporting to the correct repository (`magic-sdk`)?

### ❓ Question

On the readme for `@magic-sdk/react-native` it says that for the bare RN workflow we have to stay on version 8. And states that there will be a separate package for this case:
>Since 9.0.0, @magic-sdk/react-native package drops the support of bare React Native. You may stay on ^8.0.0 to keep your bare RN app functional. We will release a new package to support bare-RN exclusively.

Is there such a package or is it on the roadmap?

### 🌎 Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| `@magic-sdk/react-native`      | 8.2.1
| Operating System | React Native

**Comment by KevMac19** [comment]
@ilyavf I was facing the same issue in my existing running application .  

I am able to solve that by adding `expo-modules` and updating package version.

- Update package 
  ```bash
   yarn add @magic-sdk/react-native
   ```
- Install `expo-modules`
  ```bash
  npx install-expo-modules
  ```
- Android changes

   - `MainActivity.java`
   
     ```java
     package com.packagename;
     import expo.modules.ReactActivityDelegateWrapper;
     ```
   - `MainApplication.java`
   
     ```java
      package com.packagename;
      import android.content.res.Configuration;
      import expo.modules.ApplicationLifecycleDispatcher;
      import expo.modules.ReactNativeHostWrapper;
      ```
- iOS changes

  - `AppDelegate.h`
    ```h
    #import <React/RCTBridgeDelegate.h>
    #import <UIKit/UIKit.h>
    
    #import <Expo/Expo.h>
    ```

**Comment by Ethella** [comment]
It's currently on the road map. We'll announce in the thread when it's ready

**Comment by magiclabsFE** [comment]
:rocket: Issue was released in [`@magic-ext/algorand@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/algorand%408.1.0), [`@magic-ext/avalanche@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/avalanche%408.1.0), [`@magic-ext/bitcoin@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/bitcoin%408.1.0), [`@magic-ext/conflux@6.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/conflux%406.1.0), [`@magic-ext/connect@6.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/connect%406.1.0), [`@magic-ext/cosmos@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/cosmos%408.1.0), [`@magic-ext/ed25519@4.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/ed25519%404.1.0), [`@magic-ext/flow@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/flow%408.1.0), [`@magic-ext/harmony@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/harmony%408.1.0), [`@magic-ext/icon@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/icon%408.1.0), [`@magic-ext/near@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/near%408.1.0), [`@magic-ext/oauth@7.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/oauth%407.1.0), [`@magic-ext/polkadot@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/polkadot%408.1.0), [`@magic-ext/react-native-bare-oauth@7.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-bare-oauth%407.1.0), [`@magic-ext/react-native-expo-oauth@7.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-expo-oauth%407.1.0), [`@magic-ext/solana@9.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/solana%409.1.0), [`@magic-ext/taquito@5.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/taquito%405.1.0), [`@magic-ext/terra@5.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/terra%405.1.0), [`@magic-ext/tezos@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/tezos%408.1.0), [`@magic-ext/webauthn@7.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/webauthn%407.1.0), [`@magic-ext/zilliqa@8.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/zilliqa%408.1.0), [`@magic-sdk/commons@9.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/commons%409.1.0), [`@magic-sdk/pnp@7.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/pnp%407.1.0), [`@magic-sdk/provider@13.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/provider%4013.1.0), [`@magic-sdk/react-native-bare@13.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-bare%4013.1.0), [`@magic-sdk/react-native-expo@13.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-expo%4013.1.0), [`@magic-sdk/types@11.1.0`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/types%4011.1.0), [`magic-sdk@13.1.0`](https://github.com/magiclabs/magic-js/releases/tag/magic-sdk%4013.1.0) :rocket:
